### DISCUSSION TITLE
How to set background color if using FillStyleExtension

### DISCUSSION THREAD

**Discussion by zsoooc** [visgl/deck.gl#8832] [Q&A]
I use Deck GL v8.9,35. I would like to set background color for my polygon with FillStyleExtension. So, my goal is to set a background color. How to do this?
This is my code: [](https://codepen.io/vis-gl/pen/eYBJWKz)
https://codepen.io/vis-gl/pen/eYBJWKz

So, in my example I want to change the white background to red. How can I do this?

**Comment by chrisgervang** [comment]
The background isn't white, it's transparent. You can use CSS to set the background of the whole map, or if you only want to set the background within the layer, you could render two layers: one for the background w/o the extension and one for the foreground using the extension.

![Screen Shot 2024-04-25 at 11 05 05 AM](https://github.com/visgl/deck.gl/assets/2461547/486ae0bb-e794-4e0e-b83d-d0546ea9fde8)

**Reply by felixpalmer** [reply]
Fixed by https://github.com/visgl/deck.gl/pull/10618
